minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -335,15 +324,10 @@ private Process _exec(Class<?> clazz, List<String> extraJvmOpts, String... args)
         + "\n, environment: " + builder.environment());
     Process process = builder.start();
 
-    LogWriter lw;
-    lw = new LogWriter(process.getErrorStream(),
-        new File(config.getLogDir(), clazz.getSimpleName() + "_" + process.hashCode() + ".err"));
-    logWriters.add(lw);
-    lw.start();
-    lw = new LogWriter(process.getInputStream(),
-        new File(config.getLogDir(), clazz.getSimpleName() + "_" + process.hashCode() + ".out"));
-    logWriters.add(lw);
-    lw.start();
+    builder.redirectError(
```

Comment:
Can't use the builder after starting the process.

Reply:
Thanks. Working on a resolution